src/Orleans.TestingHost/TestClusterHostFactory.cs

```diff
@@ -205,7 +196,9 @@ private static void ConfigureAppServices(IConfiguration configuration, IClientBu
         private static void TryConfigureTestClusterMembership(HostBuilderContext context, IServiceCollection services)
         {
             bool.TryParse(context.Configuration["UseTestClusterMemebership"], out bool useTestClusterMemebership);
-            if (useTestClusterMemebership)
+            if (useTestClusterMemebership
+                && services.All(svc => svc.ServiceType != typeof(IMembershipTable))
+                && services.All(svc => svc.ServiceType != typeof(IMembershipOracle)))
```

Comment:
nit: could avoid double enumeration

Reply:
I'll remove this and make sure `UseTestClusterMembership` is correctly set instead